aws-sso.rb

```diff
@@ -1,12 +1,12 @@
 class AwsSso < Formula
-  desc "Tool to generate temporary AWS credentials using Okta SSO"
-  homepage "https://bitbucket.org/ensighten-ondemand/aws_sso.git"
-  version '1.0.1'
+  desc "Generate temporary AWS credentials using Okta SSO"
+  homepage "https://bitbucket.org/ensighten-ondemand/aws_sso"
 
   # This will prompt you for your Bitbucket password... kind of a pain.
   # TODO: Use git protocol?
-  url "https://bitbucket.org/ensighten-ondemand/aws_sso.git", :tag => version
-  head "https://bitbucket.org/ensighten-ondemand/aws_sso.git", :branch => 'master'
+  url "https://bitbucket.org/ensighten-ondemand/aws_sso/get/#{version}.tar.gz"
```

Comment:
Good point about the version check!

Yes, that's from within homebrew when you don't have a login cookie.

On 25 May 2016 7:41 p.m., Alejandro Figueroa <notifications@github.com> wrote:

In aws-sso.rb:

> > # This will prompt you for your Bitbucket password... kind of a pain. > # TODO: Use git protocol? > - url "https://bitbucket.org/ensighten-ondemand/aws_sso.git", :tag => version > - head "https://bitbucket.org/ensighten-ondemand/aws_sso.git", :branch => 'master' > + url "https://bitbucket.org/ensighten-ondemand/aws_sso/get/#{version}.tar.gz" 

Hmm. It downloads the .tgz in my browser. Is it a 404 from within homebrew?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub